features/scripts/build_ndk_app.sh

```diff
@@ -1,4 +1,18 @@
 #!/usr/bin/env bash
 
+BUILD_VERSION=9000.0.0-test
+
+# Build plugin
+./gradlew build -Pversion=$BUILD_VERSION
+# Install to maven local
+mvn install:install-file \
+    -Dfile=build/libs/bugsnag-android-gradle-plugin-$BUILD_VERSION.jar  \
+    -DgroupId=com.bugsnag -DartifactId=bugsnag-android-gradle-plugin \
+    -Dversion=$BUILD_VERSION \
+    -Dpackaging=jar
```

Comment:
@kattrali was there a reason `mvn` should be used rather than running `./gradlew install`?

Reply:
Not really, in the moment I thought "how should I put this in maven local? Oh I'll use maven". Installing to maven local from gradle is more consistent.